.expo/packager-info.json

```diff
@@ -1,8 +1,8 @@
 {
   "expoServerPort": 19000,
   "packagerPort": 19001,
-  "packagerPid": 37235,
-  "expoServerNgrokUrl": "https://zq-4xm.jihdeh.juli.exp.direct",
-  "packagerNgrokUrl": "https://packager.zq-4xm.jihdeh.juli.exp.direct",
-  "ngrokPid": 37249
+  "packagerPid": 2197,
+  "expoServerNgrokUrl": "https://vk-puv.johnanisere.raja.exp.direct",
```

Comment:
Okay uhm, let's add this file to .gitignore, seems it's relative to our local systems.